### PR TITLE
Add generator-gulp-angular 1.0.0 announcement blog post

### PR DESCRIPTION
**Pull request by zckrs**
Hi, it’s a great day for us (me and @Swiip) as we’re about to release the 1.0.0 of our generator. 
To announce the release and celebrate it, we have written a blog post. We mainly target our own blogs.

It crossed our mind that you would perhaps also published it directly on the Yeoman website. Of course it would mean much more visibility to us and it will be great but don’t worry and refuse the PR if you don’t feel confortable with it.

It’s also the occasion for us to tell you that we want the fountainjs project to be more involved in the yeoman community. We want more to compose with existing generators and make more modular and reusable generators.

***
TODO: edit date of blog post

### CODE REVIEW

**Comment by SBoudrias** [comment]
I'd be happy to see this posted on the Yeoman blog.

File: app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+layout: blog
+title: generator-gulp-angular 1.0.0 stable released
+excerpt: Several millions possibility to simplify your developments
+---
+
+> This is a post by [Matthieu Lux](https://github.com/Swiip) and [Mehdy Dara](https://github.com/zckrs).
```
Comment (sindresorhus) [inline app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md]: `post` => `guest post` maybe?

File: app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+We followed all the precepts of Yeoman adding our owns:
+* Gives a well written seed project following best recommendations in terms of folder structure and code style.
+* Offers lots of options enabling the user to start instantly with the best tooling and optimization adapted to the latest technologies.
+* Uses the concept of automatic injection in different parts of the project: scripts tags both vendor and sources in the index.html, styles files, vendor, css or preprocessed.
+* Provides a test coverage as perfect as possible of the code of the generator but also of the generated code.
+
+### Technologies supported
+
+We’re not joking talking of a stable version. We integrated lots of technologies and languages, from Coffee to Typescript, from Sass to Stylus. The number of combinations of our options exceeds several millions! We wrote tests, documentation and fixed issues for 12 minor versions and 2 release candidates to be able to deliver a perfectly configured seed project whatever options you choose.
```
Comment (sindresorhus) [inline app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md]: `Coffee` => `CoffeeScript`

File: app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+![](/assets/img/blog/technologies-gga.png)
+
+### Optimization served
+
+We integrate many optimizations for your web application by some plugins Gulp:
```
Comment (sindresorhus) [inline app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md]: > We integrate many optimizations for your web application using some Gulp plugins:

File: app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+* **ngAnnotate**: convert simple injection to complete syntax to be minification proof
+* **angular-templatecache**: all HTML partials will be converted to JS to be bundled in the application
+* **ESLint**: The pluggable linting utility for JavaScript
+* **watch**: watch your source files and recompile them automatically
+* **useref**: allow configuration of your files in comments of your HTML file
+* **uglify**: optimize all your JavaScript
+* **clean-css**: optimize all your CSS
+* **rev**: add a hash in the file names to prevent browser cache problems
+* **karma**: out of the box unit test configuration with karma
+* **protractor**: out of the box e2e test configuration with protractor
```
Comment (sindresorhus) [inline app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md]: Can you linkify these plugin?

File: app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+![](/assets/img/blog/technologies-gga.png)
+
+### Optimization served
+
+We integrate many optimizations for your web application by some plugins Gulp:
+* **browserSync**: full-featured development web server with livereload and devices sync
+* **ngAnnotate**: convert simple injection to complete syntax to be minification proof
+* **angular-templatecache**: all HTML partials will be converted to JS to be bundled in the application
+* **ESLint**: The pluggable linting utility for JavaScript
```
Comment (sindresorhus) [inline app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md]: lowercase `The`

File: app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+* **useref**: allow configuration of your files in comments of your HTML file
+* **uglify**: optimize all your JavaScript
+* **clean-css**: optimize all your CSS
+* **rev**: add a hash in the file names to prevent browser cache problems
+* **karma**: out of the box unit test configuration with karma
+* **protractor**: out of the box e2e test configuration with protractor
+
+## 2.0.0 on the road...
+
+But the v1 is not the end of the road. While maintaining the v1 branch, we started a new Github organization called  [FountainJS](https://github.com/FountainJS) targeting a futuristic v2 version. As the context of the build tools has greatly evolved over a year, it’ll be a reboot of the code base.
```
Comment (sindresorhus) [inline app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md]: remove double space before FountainJS link

File: app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+The major axes will be to use Yeoman generators composition, to upgrade to Gulp 4 and to write in ES6. Finally, I hope to open new horizons in terms of options: dependency management for sure, but why not also Web frameworks (someone talked about React?) and also a backend.
+
+
+Go try out [generator-gulp-angular](https://www.npmjs.com/package/generator-gulp-angular) v1.0.0 release! Any feedbacks, issues, or investment on the new  [FountainJS](https://github.com/FountainJS) project will always be appreciated.
```
Comment (sindresorhus) [inline app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md]: > Go try out the

Comment (sindresorhus) [inline app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md]: feedbacks => feedback

Comment (sindresorhus) [inline app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md]: investment => contributions

**Comment by sindresorhus** [comment]
:+1: Congrats on the release! :sparkles:

**Comment by zckrs** [comment]
:blush: 
@sindresorhus thank you for comments. All your notes are apply.

I await your approval to:
* set a date 
* squash commit

**Comment by hemanth** [comment]
@zckrs :100: Cool! :+1:

**Comment by Swiip** [comment]
Awesome, thanks a lot!

File: app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+title: generator-gulp-angular 1.0.0 stable released
+excerpt: Several millions possibility to simplify your developments
+---
+
+> This is a guest post by [Matthieu Lux](https://github.com/Swiip) and [Mehdy Dara](https://github.com/zckrs).
+
+## Intro ![Logo](/assets/img/blog/generator-gulp-angular-logo.png)
+
+
+It has now been more than a year since I ([@Swiip](https://twitter.com/Swiip)), quickly followed by [@zckrs](https://twitter.com/Zckrs), started working on our Yeoman generator. Today we’re celebrating the release of our first major and stable version : [generator-gulp-angular 1.0.0](https://www.npmjs.com/package/generator-gulp-angular).
```
Comment (eddiemonge) [inline app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md]: no space before the colon

File: app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
+---
+layout: blog
+title: generator-gulp-angular 1.0.0 stable released
+excerpt: Several millions possibility to simplify your developments
```
Comment (eddiemonge) [inline app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md]: grammar is a bit off. Not clear what the post is about

File: app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+---
+
+> This is a guest post by [Matthieu Lux](https://github.com/Swiip) and [Mehdy Dara](https://github.com/zckrs).
+
+## Intro ![Logo](/assets/img/blog/generator-gulp-angular-logo.png)
+
+
+It has now been more than a year since I ([@Swiip](https://twitter.com/Swiip)), quickly followed by [@zckrs](https://twitter.com/Zckrs), started working on our Yeoman generator. Today we’re celebrating the release of our first major and stable version : [generator-gulp-angular 1.0.0](https://www.npmjs.com/package/generator-gulp-angular).
+
+At first we simply wanted to make a good merge between [generator-gulp-webapp](https://github.com/yeoman/generator-gulp-webapp) and [generator-angular](https://github.com/yeoman/generator-angular) as I worked on Angular and got tired of Grunt verbosity. Then, the project popularity started to increase and so did its ambition.
```
Comment (eddiemonge) [inline app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md]:
```
At first, we simply wanted to make a good merge between [generator-gulp-webapp](https://github.com/yeoman/generator-gulp-webapp) and [generator-angular](https://github.com/yeoman/generator-angular), as I was working on Angular but got tired of Grunt's verbosity. Then, the project popularity started to increase and so did its ambition.
```

File: app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+
+## Philosophy
+
+We followed all the precepts of Yeoman adding our owns:
```
Comment (eddiemonge) [inline app/_posts/blog/2015-10-xx-generator-gulp-angular-v1.md]: `We followed all the precepts of Yeoman, while adding our own:`

**Comment by eddiemonge** [comment]
awesome thanks!

**Comment by eddiemonge** [comment]
https://github.com/yeoman/yeoman.github.io/commit/5d0087716ca336369d5983ebb28a00d4e8c950e1

**Pull request closed without merging**